### PR TITLE
Upgrade the following dependencies

### PR DESCRIPTION
**Pull request by rfan-debug**
## Changes

+ Upgrade scala version from 2.12 to 2.13
+ Upgrade scalapb from 0.11.6 to 0.11.10, and grpc version from 1.42.0 to 1.47.0

## How to generate the code

1. Update the `MAVEN_ARTIFACTS` and run `make rulegen`.
2. Manually update the GRPC_VERSION variable and update the hash accordingly.

### CODE REVIEW

**Comment by sheilahu** [comment]
I'm assuming that in the future if we want to update grpc version it should be relevant easy

File: repositories.bzl [hunk 1]
```diff
@@ -29,7 +29,7 @@ VERSIONS = {
         "org": "grpc",
         "repo": "grpc",
         "ref": "v{}".format(GRPC_VERSION),
-        "sha256": "b2f2620c762427bfeeef96a68c1924319f384e877bc0e084487601e4cc6e434c",
+        "sha256": "271bdc890bf329a8de5b65819f0f9590a5381402429bca37625b63546ed19e54",
```
Comment (sheilahu) [inline repositories.bzl:32]: just curious, how did you obtain those sha256? Is there a standard way

**Comment by sheilahu** [comment]
I wonder, if we can submit this as a PR to rules_proto_grpc?

**Comment by sheilahu** [comment]
if we decide to use this for aqueduct, is it possible to move into the etsy space or you prefer putting here?

File: repositories.bzl (hunk 1 above)
Comment (rfan-debug) [inline repositories.bzl:32]: I went through a pretty manual way to compute the sha256. I am wrapping up a script to automate it.

**Pull request merged**